### DISCUSSION TITLE
How could i use helix gui for wsl2 win11

### DISCUSSION THREAD

**Discussion by alexsunxl** [helix-editor/helix#4799] [Q&A] [answered]
I had try to follow the readme,but failed for this. I also try to run `wslg.exe -d ArchLinux --cd "~" -- hx` in window terminal, and it not work.
 I just use tui only in window terminal now.

There is my wsl2:

![image](https://user-images.githubusercontent.com/4388158/202648997-ec71e4a1-a20d-4895-805d-9ff87047147c.png)

So how should i do to use gui?
thanks.

**Comment by the-mikedavis** [comment, marked as answer]
The GUI is not currently implemented: https://github.com/helix-editor/helix/issues/39

For now you will need to run Helix in a terminal

**Reply by alexsunxl** [reply]
OK got that.
